src/components/CalculationResult.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+class CalculationResult extends React.Component {
+  render() {
+    const { prevNumber, operator, nextNumber } = this.props.expression;
+
+    return <h1 id="total">{prevNumber + operator + nextNumber || '0'}</h1>;
```

Comment:
number 가 포함되어있어서 + 로 스트링 붙이기 보다는, `template literals`로 해주는 게 조금 더 좋을 것 같아요 :)

```suggestion
    return <h1 id="total">{`${prevNumber}${operator}${nextNumber}` || '0'}</h1>;
```